web/src/containers/StateDashboard.js

```diff
@@ -0,0 +1,101 @@
+import PropType from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Icon, { faPlusCircle, faSpinner } from '../components/Icons';
+import Sidebar from './StateDashboardSidebar';
+import TopBtns from './TopBtns';
+import { createApd, selectApd } from '../actions/apd';
+import { Section } from '../components/Section';
+import { t } from '../i18n';
+
+const Loading = () => (
+  <div className="h2 p0 pb3 center">
```

Comment:
I know this `h2` class for text size is 😬 but I wasn't sure if we had another approach ready to go yet.

Reply:
Nope, this is fine for now (and might be fine indefinitely; since classes don't have semantic meaning, it's weird but maybe will end up being the best way to solve the problem anyway)!